src/spec-common/injectHeadless.ts

```diff
@@ -535,12 +535,15 @@ async function runLifecycleCommand({ lifecycleHook }: ResolverParameters, contai
 			if (err && (err.code === 130 || err.signal === 2)) { // SIGINT seen on darwin as code === 130, would also make sense as signal === 2.
 				infoOutput.raw(`\r\n\x1b[1m${lifecycleHookName} interrupted.\x1b[0m\r\n\r\n`);
 			} else {
+				if (err.code === 127) {
+					infoOutput.write(toErrorText(`${err.cmdOutput}`));
```

Comment:
Curious, why don't we always output the `err.cmdOutput` when there exists one ? That way even if the error code is not 127, everyone would benefit.

Reply:
done